backend/bin/deal-observer-backend.js

```diff
@@ -0,0 +1,54 @@
+import { createPgPool } from '@filecoin-station/deal-observer-db'
+import * as Sentry from '@sentry/node'
+import { ethers } from 'ethers'
+import assert from 'node:assert/strict'
+import timers from 'node:timers/promises'
+import slug from 'slug'
+import { RPC_URL, rpcHeaders } from '../lib/config.js'
+import '../lib/instrument.js'
+import {
+  observeBuiltinActorEvents
+} from '../lib/deal-observer.js'
+import { createInflux } from '../lib/telemetry.js'
+
+const { INFLUXDB_TOKEN } = process.env
+assert(INFLUXDB_TOKEN, 'INFLUXDB_TOKEN required')
+
+const pgPool = await createPgPool()
+
+const fetchRequest = new ethers.FetchRequest(RPC_URL)
+fetchRequest.setHeader('Authorization', rpcHeaders.Authorization || '')
+const provider = new ethers.JsonRpcProvider(fetchRequest, null, { polling: true })
+
+const { recordTelemetry } = createInflux(INFLUXDB_TOKEN)
+
+const loop = async (name, fn, interval) => {
```

Comment:
At this stage of the project, the `loop` function is overkill. It was obviously designed for the case where you have multiple loops. I don't want to cause any extra work though, if you don't want to refactor this could you please add a comment why this function exists?

Reply:
Very soon, we will need to add a second loop to add Payload CID to deals ingested from DDO events. Then we will need a third loop to sync DDO-sources deals to spark-api.

Having said that, @NikolasHaimerl is removing the `loop` helper in #12, so I think this comment will be resolved soon.